packages/cli/src/commands/deploy/index.ts

```diff
@@ -505,6 +508,7 @@ export default async (client: Client): Promise<number> => {
   });
   let deployStamp = stamp();
   let deployment = null;
+  const noWait = !!argv['--no-wait'];
```

Comment:
I think `!!` is unnecessary, no? It should already be a boolean.

Reply:
No problem, I'll remove in the next commit.

Side note, the `vc list` and `vc link` commands do `!!argv['--yes']`, perhaps those should also be updated?